docs/body.md

```diff
@@ -768,108 +823,84 @@ For manual configuration, you must modify every Ceilometer node as follows:
 
-            ManagePipeline: true
-            PipelinePublishers:
-              - https://10.0.0.10:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
-              - https://10.0.0.12:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
-            ManageEventPipeline: true
-            EventPipelinePublishers:
-              - https://10.0.0.10:8342/ceilometer/v1/events/myopenstack?verify_ssl=False
-              - https://10.0.0.12:8342/ceilometer/v1/events/myopenstack?verify_ssl=False
+      [oslo_messaging_notifications]
+      driver = messagingv2
+      topics = notifications
 
-*   This template must be ***rendered*** into your templates before you initiate a deployment from
-    Undercloud.<br>
-    For more information on RHOSP template management, see RedHat's
-    [Including Environment Files in Overcloud Creation](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/advanced_overcloud_customization/index#sect-Including_Environment_Files_in_Overcloud_Creation).
+#### Ceilometer Troubleshooting
 
-*   After deployment is complete, go to the undercloud, SSH into controller and go
-    to `/etc/ceilometer` in ceilometer_agent_notification container to check if
-    `pipeline.yaml` and `event_pipeline.yaml` file is updated:
+A variety of errors can be returned by zenopenstack to ceilometer.  Here are the most common ones.
 
-            ssh heat-admin@<controller ip>
-            sudo su -
-            sudo docker exec --user 0 -it ceilometer_agent_notification /bin/bash
-            # Ensure your configuration had the right publishers
+The errors would be found in `/var/log/ceilometer/agent-notification.log` on the OpenStack controller nodes.
 
-#### VM State Changes
+* Connection Refused
 
-By default, instance state changes will be captured by Zenoss when
-certain events occur, for example, when an instance is shut down, the
-state change to SHUTDOWN will be reflected in Zenoss.
+  ERROR ceilometer.pipeline.sample ConnectionError: HTTPSConnectionPool(host='1.2.3.4', port=8342): Max retries exceeded with url: /ceilometer/v1/samples/myopenstack (Caused by NewConnectionError('<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0x7f80d1e89590>: Failed to establish a new connection: [Errno 111] Connection refused',))
 
-However, certain state changes that don't correspond to another defined
-event may not be picked up until the next time Zenoss models the
-environment.
+  Verify that the IP address is correct and that the proxy-zenopenstack service is running.
 
-If you would like to reduce the likelihood of this occurring, you can
-configure OpenStack Nova to send an event (through Ceilometer) to Zenoss
-whenever any VM state change occurs by adding the following to
-/etc/nova/nova.conf on all Nova hosts:
+* Network Connectivity
 
-      [notifications]
-      notify_on_state_change=vm_and_task_state
+  ```
```

Comment:
This does't render correctly, and inserts verbatim ``` marks.
My tests show indenting by 8 chars will format it as preformatted text, again breaking lines in way that is readable.

Reply:
It looks fine to me- pandoc v2.1.3.  I then updated to 2.7.1, and it still looks fine..